### ISSUE TITLE
Can not  create new tags

### ISSUE THREAD

**Issue by xiaozai** [votchallenge/aibu#6]
When click **Metadata** to create a new tag, there is no response or error messages. So I can not create any tags.

System Info : Ubunut 16.04, Jave 1.16

**Comment by lukacu** [comment]
What do you mean by clicking on Metadata? If I remember correctly, the way to add a new tag is to write its name in the field below the tag list and press enter. But I know that it is not that intuitive.

**Comment by xiaozai** [comment]
yes, when I press enter, there is no response, and can not create tags, and there is no error messages. I guess this is also caused by Java version

**Comment by lukacu** [comment]
Why do you assume that it is a problem with the Java version? You can check the listener that should check for Enter and see if it gets triggered.

**Comment by xiaozai** [comment]
I will check it , but I am not familiar with Java
